docs/reference/http.rst

```diff
@@ -0,0 +1,218 @@
+.. _ref_reference_http_api:
+
+HTTP API
+========
+
+Using HTTP, you may check the health of your EdgeDB instance, check metrics on
+your instance, and make queries.
+
+Your instance's URL takes the form of ``http://<hostname>:<port>/``. For
+queries, you will append ``db/<database-name>/edgeql``.
+
+.. note::
+
+    Here's how to determine your development EdgeDB instance's HTTP server URL:
+
+    - The ``hostname`` will be ``localhost``
+    - Find the ``port`` by running ``edgedb instance list``. This will print a
```

Comment:
the hostname is in this output, so could pull both from there, and it would work for remote-linked instances as well.

Reply:
Do you mean that `edgedb instance list` gives you the hostname?

Reply:
yes - the "port" column is a bit misleadingly named, it's both hostname & port. here I used `edgedb instance link` on my laptop to link to an instance I have running on my desktop:

```
gold ~/src/edgedb-cli > ./target/debug/edgedb instance list
┌────────┬─────────┬──────────────┬─────────────┬────────┐
│ Kind   │ Name    │ Port         │ Version     │ Status │
├────────┼─────────┼──────────────┼─────────────┼────────┤
│ remote │ desktop │ cobalt:15656 │ 2.9+5ac35ff │ up     │
└────────┴─────────┴──────────────┴─────────────┴────────┘
gold ~/src/edgedb-cli > xh cobalt:15656/server/status/ready
HTTP/1.1 200 OK
Content-Length: 4
Content-Type: application/json

"OK"
```

Reply:
Ah, OK. So, for remote instances, you'll also get the hostname in the "Port" column.

Should I include this in a new paragraph in this note addressing production instances, or is there a case where someone would have a remote development instance? If this primarily addresses production, I could add something like this:

> To determine the URL of a production instance, you can get both the hostname and port of the instance from the "Port" column of the `edgedb instance list` table (formatted as `<hostname>:<port>`) if you have already linked to the instance. The same guidance on development database names applies here.

Reply:
AFAIK remote instances can be either production or development - I can see a remote dev instance being useful in that laptop/desktop case, and a remote production instance being useful for debugging.

Reply:
I'd love to get your thoughts, @zackelan, on this rework. Changed "development" in the original to "local" and added a paragraph for determining remote instance URL to incorporate your suggestion.

Thanks for the great feedback, by the way!

Reply:
LGTM!